### DISCUSSION TITLE
Anyone know which component to create muli level dropdown menu like this?

### DISCUSSION THREAD

**Discussion by kompany-sde** [shadcn-ui/ui#1357] [General]
Hi Guys, 

Can someone help me create an accessible dropwdown multt-level menu like in the image below?  This can be seen on the product hunt webite when you hover over categories.

![image](https://github.com/shadcn-ui/ui/assets/77177020/f6eb805b-fd8a-4577-bfcc-ca6a2e0b7616)

**Comment by kellymweu** [comment]
![Shadcn-DropDown menu](https://github.com/shadcn-ui/ui/assets/13185391/bfb4c3ca-0b15-4e49-ab1b-f92770577bd0)
Hello Kompany-sde,
Kindly try customizing the DropDown Menu Component to create 2 extra columns, for a total of 3 as in your screenshot.
Cheers. : )

**Reply by kompany-sde** [reply]
Thanks. Will give this a try
